### PR TITLE
XW-518 | ES6 server-side

### PR DESCRIPTION
**Pull request by vforge**
https://wikia-inc.atlassian.net/browse/XW-518

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,29 +3,28 @@
...
-        localSettings = baseLocalSettings.extendSettings({
-            devboxDomain: 'joe',
-            loggers: {
-                console: 'debug'
-            },
-            port: 8000 // 7000 if running on devbox
-    });
-
-    export = localSettings;
+	import baseExtendSettingsDev from './localSettings.dev';
```
Comment (rogatty) [inline README.md]: It would look better without tab.

File: README.md [hunk 2]
```diff
@@ -3,29 +3,28 @@
...
+	import baseExtendSettingsProd from './localSettings.base';
+
+	const baseExtendSettings = process.env.WIKIA_ENVIRONMENT === 'dev' ? baseExtendSettingsDev : baseExtendSettingsProd,
+		localSettings = baseExtendSettings({
+			devboxDomain: 'joe',
+			loggers: {
+				console: 'debug'
+			},
+			port: 8000 // 7000 if running on devbox
```
Comment (rogatty) [inline README.md]: I think we should just leave 7000 as devboxes are now the default.

File: server/lib/Utils.js [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+}
+
+/**
+ * Get environment as string
+ *
+ * @param {String} environment
+ * @returns {string}
+ */
+export function getEnvironmentString(environment) {
+	return getEnvironmentString(environment).toLowerCase();
```
Comment (rogatty) [inline server/lib/Utils.js]: This seems strange, maybe it should be `getEnvironment(environment)`?

File: config/authLocaleSettings.js [hunk 4]
```diff
@@ -1,7 +1,9 @@
-var deepExtend: any = require('deep-extend');
+/* eslint dot-notation:0 */
 
-var localeSettings: any = {
-	'en' : {
+import deepExtend from 'deep-extend';
+
+const authLocaleSettings = {
```
Comment (kvas-damian) [inline config/authLocaleSettings.js]: Don't you think that this file should be shared across entire mercury platform?
In that case ```auth``` prefix doesn't make sense here.

**Pull request merged**